### DISCUSSION TITLE
page content disappears

### DISCUSSION THREAD

**Discussion by DHS1001** [docmost/docmost#459] [Help / Questions] [answered]
In the latest 0.5.0 version, the issue is not resolved.
When changing the content of the page, it is temporory save, but the time stamp is not updated. 
The added content disappears after some time.

It is the same issue as mentioned in https://github.com/docmost/docmost/issues/308#issuecomment-2415370967

Please find more details via the below screen capture.
https://github.com/user-attachments/assets/0961062d-7225-4796-ae68-acc0587b3211

The instance is run via docker compose in a Synology NAS.

**Comment by Philipinho** [comment]
Did you upgrade from a previous version?

**Reply by DHS1001** [reply]
Yes, the instance was upgraded from the 0.4.1

**Reply by DHS1001** [reply, marked as answer]
Did a fresh installation of the 0.5, and it is working properly now.
Thanks @Philipinho

**Comment by nobrainer-de** [comment]
Hey I got the same issue, different tries with nginx reverse settings, web socket is enabled.
Fresh installation of 0.5, docker compose on a vserver (Debian).

Yesterday it worked quiet good for 2 pages, but today only new page creation or editing name of the page is being synced. The site data just stays for every session only locally.

The date of editing pages is being synced correctly (timestamp is correct).

**Reply by Philipinho** [reply]
Please confirm if there are any websocket connection errors in your browser console.

**Reply by nobrainer-de** [reply]
Nevermind, it was full okay, Client 02 was behind a Zscaler connection, there is some caching problem.
With 2 other devices not behind zscaler, it works fine. Sorry!
